openapi/MainResources-local-openapi-v3.yml

```diff
@@ -464,31 +462,31 @@ components:
         allDoors:
           type: array
           items:
-            type: object
+            $ref: '#/components/schemas/Window'
         openDoors:
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'  
         unknownDoors:
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'
         allWindows:
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'
         openWindows:
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'
         unknownWindows:
           type: array
           items:
-            type: object        
+            $ref: '#/components/schemas/Window'
         allOthers:
```

Comment:
`allOthers` should not be available anymore

Reply:
On HomeController I (newest version) it still is
![image](https://github.com/BoschSmartHome/bosch-shc-api-docs/assets/20702171/e0938476-b163-4455-a304-01312e1b0c1a)